src/HeaderCell.tsx

```diff
@@ -9,7 +9,7 @@ import ColumnResizeHandler, { FixedType } from './ColumnResizeHandler';
 import { useUpdateEffect, useClassNames } from './utils';
 import Cell, { InnerCellProps } from './Cell';
 
-export interface HeaderCellProps extends InnerCellProps {
+export interface HeaderCellProps extends Omit<InnerCellProps, 'onResize'> {
```

Comment:
But there is no definition of `onResize` in `InnerCellProps`

Reply:
Yes, it's in the React: `React.DOMAttributes<T>.onResize?: ReactEventHandler<T> | undefined`.

`interface StandardProps extends React.HTMLAttributes<HTMLElement>` inherits that method

Reply:
@simonguo any update?

Reply:
Fixed in [rsuite-table@5.11.1](https://github.com/rsuite/rsuite-table/releases/tag/5.11.1)